test/mouse_test.py

```diff
@@ -56,9 +56,27 @@ def test_set_pos__invalid_pos(self):
             with self.assertRaises(TypeError):
                 pygame.mouse.set_pos(invalid_pos)
 
-    def todo_test_set_pos(self):
-        """Ensures set_pos works correctly."""
-        self.fail()
+    def test_set_pos(self):
```

Comment:
Looks like the dummy video driver/window manager used on CI doesn't generate mouse events at all. That's OK though, we can skip this test if the video driver is a "dummy" like so:

```suggestion
    @unittest.skipIf(
            os.environ.get("SDL_VIDEODRIVER") == "dummy",
            'mouse events require a non-"dummy" SDL_VIDEODRIVER',
    )
    def test_set_pos(self):
```

Reply:
I suppose a clock timeout might also be good, so we don't ever get into an endless loop like:

    clock = pygame.time.Clock()
    for x, y in expected_pos:
        pygame.event.get()

        pygame.mouse.set_pos(x, y)
        ready_to_test = False
        clock.tick()
        time_passed = 0.0
        while not ready_to_test and time_passed < 1000.0:
            time_passed += clock.tick()